### ISSUE TITLE
Is it possible to import only one list?

### ISSUE THREAD

**Issue by LeoLuizGO** [TobiasPankner/Letterboxd-to-IMDb#8]
I mean, I already have some lists of letterboxd on imdb, but i want add one more from letterboxd, is it possible to import just this list?

**Comment by TobiasPankner** [comment]
No, importing lists other than the watchlist is currently not possible. I'll keep it in mind for the future but don't expect it too soon.
Thanks for the suggestion
